### ISSUE TITLE
In reaction view, automatically make a table with all the relevant H and P statements. 

### ISSUE THREAD

**Issue by kjappelbaum** [cheminfo/eln.epfl.ch#155]
In http://www.cheminfo.org/Chemistry/Cheminformatics/Swiss_GHS_-_CLP_safety/index.html you actually have already the tools to do this. And in many labs (in all labs in my undergrad institution) you need to compile a list of H/P statements before  you start the reaction.

**Comment by kjappelbaum** [comment]
https://iuclid6.echa.europa.eu/public-api

**Comment by lpatiny** [comment]
The current result does not match aldrich information for triethylamine

![image](https://user-images.githubusercontent.com/1484241/106006434-80bab980-60b5-11eb-9c41-9bf5b7b3b498.png)

Here is aldrich:

![image](https://user-images.githubusercontent.com/1484241/106006471-8ca67b80-60b5-11eb-9388-a03109783b04.png)

I guess this could be related to many different concentration and we should be able to have a detailed view with all the products (and concentration) for which GHS was provided.

**Comment by lpatiny** [comment]
The same is true for isobutyryl chloride. From Aldrich we have

![image](https://user-images.githubusercontent.com/1484241/106006705-ce372680-60b5-11eb-95f5-0547fe2e1859.png)

**Comment by kjappelbaum** [comment]
Yes, we return by default in the summary the maximum number of pictograms (which makes sense from my point of view, better safe than sorry) and then different sources have different number of pictograms https://pubchem.ncbi.nlm.nih.gov/compound/8471#section=GHS-Classification&fullscreen=true
